plugins/single_plugins/idle.cpp

```diff
@@ -218,20 +220,28 @@ class wayfire_idle_plugin : public wf::per_output_plugin_instance_t
             has_fullscreen = toplevel->pending_fullscreen();
         }
 
+        wf::get_core().connect(&inhibit_changed);
+
+        on_seat_activity = [=] (void*)
+        {
+            create_screensaver_timeout();
+        };
+
+        if (disable_initially)
+        {
+            return;
+        }
+
         update_fullscreen();
```

Comment:
I think we still need to call update_fullscreen even if it is initially disabled, so that we can properly keep track of the state (if i remember enough about the plugin, it is one of the plugins I have least idea about how it works).

Reply:
actually it doesn't seem to matter, reading the code. Thanks :)